app/src/models/conflicts.ts

```diff
@@ -10,8 +10,10 @@ import { GitStatusEntry, UnmergedEntry } from './status'
 export type ConflictFileStatus =
   | {
       readonly kind: 'text'
-      /** This number should be greater than zero */
-      readonly conflictMarkerCount: number
+      /** This number should be greater than zero, or null
```

Comment:
:art: We usually start the text of a multi-line comment after the first line to give it some space to breathe around the code